docker/Dockerfile.python2

```diff
@@ -9,27 +9,52 @@
 FROM ubuntu:18.04
 
 RUN apt-get update
-RUN apt-get install -y software-properties-common
+RUN apt-get install -y software-properties-common apt-utils
 RUN add-apt-repository -y ppa:nicotine-team/stable
-RUN apt-get install -y nicotine xfce4-terminal
+RUN apt-get install -y nicotine xfce4-terminal less
 
 COPY docker/config /root/.nicotine
 
 ENV slskproto="/usr/lib/python2.7/dist-packages/pynicotine/slskproto.py"
 # run()
 RUN sed -i '/if connection in conns and len(conns\[connection\].ibuf) > 0:/a\\
 \t\t\t\t\tprint "********    ibuf: ", conns[connection].ibuf' $slskproto
+RUN sed -i '/self.process_server_input(conns\[server_socket\].ibuf)/a\\
```

Comment:
@lene this looks great. If I understand this correctly, this is a unit test simulating communication with the server. If you get a chance, try and find a way to use this from DEP-8 as well. We have a basic stub in debian/tests/control already.

Reply:
> If I understand this correctly, this is a unit test simulating communication with the server. 

Sorry, no, this is humble printf-debugging. But actually I _did_ start o work on a mock server, check out https://github.com/lene/nicotine-plus/blob/test_suite/tests/mock_socket.py if you want to see what I did until now.
I'm torn between how to invest my limited time best at the moment. I'm still stuck at search (the client does not seem to receive any `FileSearchResult`s) and I can't decide whether I should do the debugging right (i.e. write a test suite and use it to zero in on the problem, which is a pretty massive time investment up front) or simple (i.e. printf debugging, which is ineffective and may well be more work in the end than writing a test suite). Whenever I have some time to work on nicotine+, I choose whatever I fell like at the moment, so I'm actually half-assing both approaches. Not super efficient but meh, whatever :)

> If you get a chance, try and find a way to use this from DEP-8 as well. We have a basic stub in debian/tests/control already.

That's nice, I had no idea you could do that! I'll check it out once I have a functional test suite. In the meantime I have set up a GitHub action to run the test suite on every push to my unfinished `test_suite` branch: https://github.com/lene/nicotine-plus/blob/test_suite/.github/workflows/pythonapp.yml